com.unity.cinemachine/Documentation~/samples-tutorials.md

```diff
@@ -2,15 +2,48 @@
+
+| Sample scene | Context | Use cases and key features |
+| :--- | :--- | :--- |
+| **2D Fighters** | Four-player fighting game (simulation). | <ul> <li>Set up a camera that dynamically composes the shot according to the remaining players in the game.</li> <li>Emulate players that move along predefined paths.</li> </ul> |
+| **2D Platformer** | 2D Platformer game with boss room. | <ul> <li>Set up a custom camera group that switches cameras based on the player state.</li> <li>Set up a camera that frames a boss room entirely when the player enters the room.</li> <li>Confine the camera to prevent it from seeing outside the game map.</li> </ul> |
+| **Camera Magnets** | 2D Platformer game with framing variations. | <ul> <li>Set up "magnets" that attract the camera's attention according to the player's proximity.</li> <li>Confine the camera to prevent it from seeing outside the game map.</li> </ul> |
+
+### 3D Samples
+
+Once you import the 3D Samples set, the following scenes are available in the `Assets/Samples/Cinemachine/<version number>/3D Samples` folder.
+
+| Sample scene | Context | Use cases and key features |
+| :--- | :--- | :--- |
+| **Brain Update Modes** | Car racing game (simulation). | <ul> <li>Set up the Cinemachine Brain and the animated objects in the camera composition to get a synchronized render of the whole.</li> </ul> |
+| **Clear Shot** | Third-person follow from various perspectives. | <ul> <li>Set up a group of fixed and moving cameras that follow the same player from different perspectives.</li> <li>Get the camera automatically switch to the best available shot when the current camera looses sight of the player.</li> </ul> |
+| **Custom Blends** | Third-person follow with character switching. | <ul> <li>Set up a custom camera blending algorithm.</li> <li>Map the custom algorithm to a Cinemachine Brain event.</li> </ul> |
+| **Cutscene** | Third-person follow with treasure area. | <ul> <li>Set up a cutscene with camera blends and asset animations in Timeline.</li> <li>Trigger a cutscene that blends in from the game camera, plays through, and then blends back out to the game camera.</li> </ul> |
+| **Fly around** | First-person flight simulator. | <ul> <li>Set up a fly-around camera with basic height and speed controls. |
+| **FreeLook Deoccluder** | Third-person free look with obstacle handling. | <ul> <li>Set up a free look camera that handles occlusion by walls to keep the player in view.</li> <li>Set up the scene and the camera to consider certain objects as transparent and ignore them in the occlusion evaluation.</li> </ul> |
+| **FreeLook on Spherical Surface** | Third-person free look with local up direction. | <ul> <li>Set up a free look camera to follow a player that walks on the surface of a sphere.</li> <li>Set up the camera to follow the character either lazily or actively.</li> </ul> |
```

Comment:
Camera automatically re-orients to keep the player upright.

Reply:
I will rephrase with that wording, thanks!